docs/preview/connected-registry/connected-registry-error-codes.md

```diff
@@ -0,0 +1,86 @@
+
+## Status Details Format
+
+When your connected registry has a connection state of `Unhealthy` you can run the `az acr connected-registry show` command to view the statusDetails.
+
+`StatusDetails` provides a list of errors, each with the following format:
+
+```json
+{
+  "code": "Error code",
+  "correlationId": "CorrelationId of the error on the on-premises connected registry instance",
+  "description": "Description corresponding to this error",
+  "timestamp": "Timestamp corresponding to this error",
+  "type": "Component of the connected registry instance corresponding to the error"
+}
+```
+
+Every time the connected registry instance syncs with the cloud, these status details are updated. When the connected registry no longer has status details listed, it is considered healthy and its connection state is transitioned from `Unhealthy` to `Online`. Once the connected registry is no longer connected to internet, its connection state will transition to `Offline`.
+
+## Error Code Reference
+
+This section lists the possible codes you may see in the `StatusDetails` property of a connected registry, which indicate critical errors. For each error, possible solutions are listed.
+
+### DiskError
+
+This is the default error code when the connected registry is unable to create, write, or delete a file on the local disk. There are a few scenarios that may cause a `DiskError` code. Please reference below for possible scenarios and remediations. 
+
+1. The connected registry was unable to write any file to the disk because it did not have sufficient permissions.
+
+Sample status description: "Access to the path '/var/acr/data/registry/dummy.txt' is denied."
+
+*Potential solution:* Ensure that the host storage path used to run the connected registry container gives sufficient access to the container user. In the sample above, this path is `/var/acr/data/registry`. Update the permissions of the host system directory so that the user profile for your container has read, write, and execute access. By default, docker containers run as root. If the container is run as a non-root user, please ensure that user has the above permissions.
+
+2. The connected registry was unable to write files to the local host because there was no storage available on disk.
+
+Sample status description: "There is not enough space on the disk."
+
+*Potential solution:* Connected registry container logs are integrated with docker. By default, docker does not set container log size limits. Over time, these logs can take up much of your host's storage capacity. If your disk is out of space then you can place limits on the container logs retained by docker.
+
+#### Option 1: Place global log limits for all containers on the host
```

Comment:
Are those solutions for the second problem where there is not enough space? If so I would format the doc differently using hierarchical headings instead.

As formatted now, I will assume that those solutions are for all disk problems but they sound to me more applicable only for the out of space one.

Reply:
Yes, these solutions are only for issues where the user is out of disk space

Reply:
Will adjust the format

Reply:
Please see fix